src/Subject.ts

```diff
@@ -123,6 +123,12 @@ export class Subject<T> extends Observable<T> implements Observer<T>, Subscripti
     this._complete();
   }
 
+  asObservable(): Observable<T> {
+    return new Observable((observer: Observer<any>) => {
```

Comment:
Actually, just noticed `source` is protected member so can't assign it directly. exposing it as public member seems not legit - what do you think?